README.md

```diff
@@ -33,14 +33,15 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your Team Lead
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
-
+Semantic HTML is a way to use meaningful element tags that convey information about the content.
 2. Name two big differences between ```display: block;``` and ```display: inline;```.
-
+'display: block;' takes up 100% of available width, and always starts a new line. 'display:inline' Is the opposite; it takes up the minimum width, and does not start a new line.
 3. What are the 4 areas of the box model?
-
+Outside-in: margin, border, padding, content
 4. While using flexbox, what axis does the following property work on: ```align-items: center```?
-
+'align-items' works on the vertical axis of the parent class.
 5. Explain why git is valuable to a team of developers.
+Git is valuable for developer teams because it allows multiple users to share source code and make changes freely without automatically changing the master code. The team can then evaluate all changes before merging the new code into the existing main.
```

Comment:
Great Definition! Git is a powerful tool that has become industry standard, as you progress through lambda you will use it so much that it will become second nature.